auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/SessionHolder.java

```diff
@@ -26,15 +26,17 @@
  * Created by AR288503 on 9/6/2015.
  */
 public class SessionHolder {
+
     public static final int MAX_INACTIVE_SESSION_DURATION = 600;
+    private static Map<String, AuthResult> authResultMap = new HashMap<String, AuthResult>();
+
+    private SessionHolder(){}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)